### PR TITLE
Merging template updates 2.3.1

### PR DESCRIPTION
**Pull request by d4straub**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/mag/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/mag _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: nextflow_schema.json [hunk 1]
```diff
@@ -163,6 +164,15 @@
                     "fa_icon": "fas fa-question-circle",
                     "hidden": true
                 },
+                "publish_dir_mode": {
```
Comment (jfy133) [inline nextflow_schema.json:167]: Something to update in `modules.conf` already?

**Review by jfy133 — APPROVED** [review]
One little comment, but very likely would be worth updating wholesale when all the local modules are replaced with the nf-core 'offiical' ones

File: nextflow_schema.json (hunk 1 above)
Comment (d4straub) [reply nextflow_schema.json:167]: Updated it for all modules

**Comment by d4straub** [comment]
I'll merge that and make a separate PR for 2.3.2!

**Pull request merged**